modules/apps/static/portal-configuration/portal-configuration-metatype-api/src/test/java/com/liferay/portal/configuration/metatype/bnd/util/ConfigurableUtilTest.java

```diff
@@ -225,7 +225,8 @@ private void _assertTestConfiguration(
 		Assert.assertEquals(1, testConfiguration.testShort());
 		Assert.assertEquals("test_string", testConfiguration.testString());
 		Assert.assertEquals(
-			"a=b\\,c= d", testConfiguration.testStringBackslashEscape());
+			"a\\=b\\\\\\,c\\=\\ d",
```

Comment:
@marianoalvarosaiz If your expected result is `"a\\=b\\\\\\,c\\=\\ d"`, how would I escape `=` if I want it to be part of my value?

Tina.